frontend/src/helpers/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const BASE_API_URL="http://localhost:8080";
+
+export const LOCAL_STORAGE_API_KEY = 'token';
+
+export const METHOD = {

```

Comment:
there are multiple methods in the object so call the variable _methods_ or even _HTTP_METHODS_

Reply:
Done